### PR TITLE
Add `metaimageio` from PyPI

### PR DESCRIPTION
**Pull request by auneri**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/metaimageio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - imageio >=2.8
+    - numpy >=1.19
+    - python >=3.6
+
+test:
+  imports:
+    - metaimageio
+
+about:
+  home: https://auneri.github.io/metaimageio
```
Comment (dopplershift) [inline recipes/metaimageio/meta.yaml:33]: I think GitHub URLs are case-senstive:
```suggestion
  home: https://auneri.github.io/MetaImageIO
```
(old one was 404-ing)
```suggestion
  home: https://auneri.github.io/metaimageio
```

File: recipes/metaimageio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+{% set hash = "fa1e0aa1980e10aac3ceb0bfa794b734413f1399c750fd8208dcb2f869ed37df" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
```
Comment (dopplershift) [inline recipes/metaimageio/meta.yaml:12]: Go ahead and just put the hash in here directly and don't bother with templating it--it's not used in multiple places (ever).

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor changes and this is good to go.

**Comment by auneri** [comment]
All done, thanks @dopplershift for the review

**Review by dopplershift — APPROVED** [review]

**Pull request merged**